.travis.yml

```diff
@@ -1,29 +1,42 @@
-# Linux Build Configuration for Travis
-
 language: cpp
 
 os:
   - linux
 
 # Use Ubuntu 14.04 LTS (Trusty) as the Linux testing environment.
-sudo: required
 dist: trusty
+sudo: false
 
-env:
-  # Each line is a set of environment variables set before a build.
-  # Thus each line represents a different build configuration.
-  - BUILD_TYPE=Release
-  - BUILD_TYPE=Debug
+git:
+  depth: false
+
+addons:
+  apt:
+    sources:
+    - llvm-toolchain-trusty
```

Comment:
It might be useful add a comment that it is an alias, and what exactly repository it adds.